File: days_mobile/lib/models/AccountMovement.dart
class AccountMovement {
  int id;
  Null userId;
  int studentId;
  int amount;
  String isDebt;
  String location;
  String createdAt;

  AccountMovement(
      {this.id,
      this.userId,
      this.studentId,
      this.amount,
      this.isDebt,
      this.location,
      this.createdAt,});

  AccountMovement.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    userId = json['user_id'];
    studentId = json['student_id'];
    amount = json['amount'];
    isDebt = json['is_debt'];
    location = json['location'];
    createdAt = json['created_at'];

  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['user_id'] = this.userId;
    data['student_id'] = this.studentId;
    data['amount'] = this.amount;
    data['is_debt'] = this.isDebt;
    data['location'] = this.location;
    data['created_at'] = this.createdAt;

    return data;
  }
}

File: days/app/Http/Resources/AccountMovement.php
<?php

namespace App\Http\Resources;

use Illuminate\Http\Resources\Json\JsonResource;

class AccountMovement extends JsonResource
{
    /**
     * Transform the resource into an array.
     *
     * @param  \Illuminate\Http\Request  $request
     * @return array
     */
    public function toArray($request)
    {
        return [
            'user_id' => $this->user_id,
            'student_id' => $this->student_id,
            'amount' => $this->amount,
            'is_debt' => $this->is_debt,
            'location' => $this->location,
        ];
    }
}
